.github/workflows/CODEOWNERS

```diff
@@ -0,0 +1 @@
+*       @canonical/telco
```

Comment:
There's already a codeowners file + this one points to telco (instead of TLS)

Reply:
oh, looked for it in `.github` dir :/
removed.